server/cluster/cluster.go

```diff
@@ -1781,3 +1784,42 @@ func newCacheCluster(c *RaftCluster) *cacheCluster {
 		stores:      c.GetStores(),
 	}
 }
+
+// Evict the leaders when the store is damaged. Damaged regions are emergency errors
+// and requires user to manually remove the `evict-leader-scheduler` with pd-ctl
+func (c *RaftCluster) handleDamagedStore(stats *pdpb.StoreStats) {
+	// ToDO: regions have no special process for the time being
+	// and need to be removed in the future
+	damaged_regions := stats.GetDamagedRegionsId()
```

Comment:
What is the definition of the damaged region? Is there any issue related to this PR?